scout/constants/__init__.py

```diff
@@ -95,8 +95,10 @@
     "variant",
     "acmg",
 ]
+GENOME_BUILD_37 = "37"
+GENOME_BUILD_38 = "38"
+BUILDS = [GENOME_BUILD_37, GENOME_BUILD_38, "GRCh38"]
```

Comment:
Who knows why here we had also GRCh38.. Perhaps it's needed to visualize the alignments?

Reply:
No idea, but the RNA view seems like the place to start looking?